src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule72thesummaryfragment/InputIncorrectSummaryFragment.java

```diff
@@ -4,17 +4,19 @@
  * A {@code Foo. Foo}
  * is a simple Javadoc. Some javadoc.
  */
-class InputIncorrectSummaryJavaDocCheck {
+class InputIncorrectSummaryFragment {
 
 /*warn*//**
      * As of JDK 1.1, replaced by {@link #setBounds(int,int,int,int)}
      */
     void foo3() {}
+    // violation 4 lines above 'First sentence of Javadoc is missing an ending period.'
 
```

Comment:
Please do `violation below` for all, the closer comment the better

Reply:
done